Makefile

```diff
@@ -1,14 +1,14 @@
 
 .EXPORT_ALL_VARIABLES:
 .NOTPARALLEL:
-.ONESHELL:
```

Comment:
removed so that if an individual make command fails, the entire make target fails